website/public/js/services/analyticsServices.js

```diff
@@ -119,17 +119,19 @@ function analyticsFactory(User) {
   }
 
   function updateUser(properties) {
-    if (typeof properties === 'undefined') properties = {};
```

Comment:
Not really. That was the result of my flailing around the errors resulting from the user not being available to the test.